janusgraph-core/src/main/java/org/janusgraph/graphdb/configuration/GraphDatabaseConfiguration.java

```diff
@@ -704,7 +704,8 @@ public boolean apply(@Nullable String s) {
      */
     public static final ConfigOption<Boolean> IDS_FLUSH = new ConfigOption<>(IDS_NS,"flush",
             "When true, vertices and edges are assigned IDs immediately upon creation.  When false, " +
-            "IDs are assigned only when the transaction commits. Must be disabled for graph partitioning to work.",
+            "IDs are assigned only when the transaction commits.",
+//            Must be disabled for graph partitioning to work.",
```

Comment:
consider just removing the comment since each graph necessarily has 2 virtual partitions at least.